infrastructure-playbooks/fs_status.yml

```diff
@@ -0,0 +1,15 @@
+- name: get fsmap
+  command: ceph --cluster {{ cluster }} fs dump --format=json
+  delegate_to: "{{ groups[mon_group_name][0] }}"
+  register: get_fsmap
+  when: not containerized_deployment
+
+- name: get fsmap (containerized)
```

Comment:
Isn't it possible to merge this task with the one above using `{{ docker_exec_cmd }}` logic like we do at many places in the playbook?

Reply:
Maybe? I have no idea. I'm just following code I copy-pasted from somewhere else.